--- before/patlab.py	2012-06-08 18:30:14.388333000 -0400
+++ after/patlab.py	2012-06-08 18:54:18.058006487 -0400
@@ -94,15 +94,6 @@
 		self.diffs = []
 		self.name  = name
 
-	def normalize( self ):
-		self.diffs = filter( lambda d: not d.is_identity(), self.diffs )
-		self.diffs_by_lname  = {}
-		self.diffs_by_rname = {}
-		for diff in self.diffs:
-			self.diffs_by_lname [ diff.lname ] = diff
-			self.diffs_by_rname [ diff.rname ] = diff
-		return self
-
 	def identifier( self ): return self.name
 	def elements( self ): return self.diffs
 
@@ -134,6 +125,15 @@
 				result.diffs.append( other.diffs_by_lname[ name ] )
 		return result.normalize()
 
+	def normalize( self ):
+		self.diffs = filter( lambda d: not d.is_identity(), self.diffs )
+		self.diffs_by_lname  = {}
+		self.diffs_by_rname = {}
+		for diff in self.diffs:
+			self.diffs_by_lname [ diff.lname ] = diff
+			self.diffs_by_rname [ diff.rname ] = diff
+		return self
+
 	def write_contents_to( self, file ):
 		for diff in self.diffs:
 			diff.write_contents_to( file )
