--- before/patlab.py	2012-06-09 15:10:17.607285000 -0400
+++ after/patlab.py	2012-06-09 15:10:43.456625893 -0400
@@ -628,7 +628,7 @@
 		# in the final patch.  Use shrinkwrapped() to trim unwanted
 		# context lines.
 
-		while self_feeder and other_feeder: # conflict-causing change
+		while self_feeder and other_feeder: # changed again
 			debug( "shadow", "Feeders: %3d|%-3d %3d|%-3d", self_feeder.line_number1, self_feeder.line_number2, other_feeder.line_number1, other_feeder.line_number2 )
 			debug( "shadow", "  self top pair: %s", self_feeder.top_pair() )
 			if self_feeder.line_number2 < other_feeder.line_number1:
